src/lib/operators/join_sort_merge.cpp

```diff
@@ -893,6 +1223,49 @@ class JoinSortMerge::JoinSortMergeImpl : public AbstractReadOnlyOperatorImpl {
         _output_pos_lists_right.push_back(std::move(null_output_right));
       }
     }
+
+    if (_mode == JoinMode::AntiNullAsTrue) {
```

Comment:
Can you add a sentence about the rows that are not NULLs?

Reply:
How exactly do you mean that?